source/jormungandr/jormungandr/parking_space_availability/bss/atos.py

```diff
@@ -37,12 +38,11 @@
 
 class AtosProvider(BssProvider):
 
-    OPERATOR = 'Keolis'
-
-    def __init__(self, id_ao, network, url, timeout=5):
+    def __init__(self, id_ao, network, url, operators, timeout=5):
         self.id_ao = id_ao
         self.network = network
         self.WS_URL = url
+        self.operators = map(unicode.lower, operators)
```

Comment:
isn't it better to store them as lower ?
and in this case you can store the network as lower too